src/storage/btree.c

```diff
@@ -24840,11 +24841,18 @@ btree_range_scan_descending_fix_prev_leaf (THREAD_ENTRY * thread_p, BTREE_SCAN *
       return NO_ERROR;
     }
   /* Conditional latch failed. */
-
+  /* The page can also be null due to page desyncronization only on PTS, in this case the scan must be restarted */
+  if (er_errid () == ER_PAGE_AHEAD_OF_REPLICATION)
+    {
+      bts->force_restart_from_root = true;
+      pgbuf_unfix_and_init (thread_p, bts->C_page);
+      return NO_ERROR;
+    }
   /* Unfix current page and retry. */
   pgbuf_unfix_and_init (thread_p, bts->C_page);
   error_code =
-    pgbuf_fix_if_not_deallocated (thread_p, &prev_leaf_vpid, PGBUF_LATCH_READ, PGBUF_UNCONDITIONAL_LATCH, &prev_leaf);
+    pgbuf_fix_if_not_deallocated_with_repl_desync_check (thread_p, &prev_leaf_vpid, PGBUF_LATCH_READ,
+							 PGBUF_UNCONDITIONAL_LATCH, &prev_leaf);
```

Comment:
Add a comment in the scope
```
  if (prev_leaf == NULL)
    {
      /* deallocated */
      bts->force_restart_from_root = true;
      return NO_ERROR;
    }
```
below that the restart can also happen due to desynchronization